dev_tools/modules.py

```diff
@@ -0,0 +1,167 @@
+# Copyright 2021 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import dataclasses
+import os
+import sys
+from pathlib import Path
+from typing import List, Dict, Any
+
+_FOLDER = 'folder'
+_PACKAGE_PATH = 'package-path'
+
+
+@dataclasses.dataclass
+class Module:
+    root: Path
+    raw_setup: Dict[str, Any]
+
+    name: str = dataclasses.field(init=False)
+    version: str = dataclasses.field(init=False)
+    top_level_packages: List[str] = dataclasses.field(init=False)
+    # top_level_package_paths: List[Path] = dataclasses.field(init=False)
```

Comment:
Remove comment.

Reply:
oh, actually I should uncomment it, it was an experiment whether mypy detects the issue that we are setting an undeclared in `__post_init__` (it doesn't). Adding it back!